python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -11,43 +11,45 @@
 
 
 class WLSDetailedState(enum.IntEnum):
```

Comment:
Consider renaming this to be more representative of what it actually is, e.g.; `KiloArcState`.

Also, consider publishing this information as an event.

Reply:
saving this one for subsequent ticket